examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/SW4STM32/STM32F401RE-Nucleo/xively_demo_stm32f4xx-nucleo-wifi/.cproject

```diff
@@ -111,6 +112,127 @@
 						</toolChain>
 					</folderInfo>
 					<fileInfo id="fr.ac6.managedbuild.config.gnu.cross.exe.debug.1474710539.110071304" name="LinkerScript.ld" rcbsApplicability="disable" resourcePath="LinkerScript.ld" toolsToInvoke=""/>
+					<sourceEntries>
+						<entry excluding="Drivers/BSP/Components/lis2dh12|Drivers/BSP/Components/h3lis331dl|Drivers/BSP/X_NUCLEO_IKS01A2" flags="VALUE_WORKSPACE_PATH|RESOLVED" kind="sourcePath" name=""/>
+					</sourceEntries>
+				</configuration>
+			</storageModule>
+			<storageModule moduleId="org.eclipse.cdt.core.externalSettings"/>
+		</cconfiguration>
+		<cconfiguration id="fr.ac6.managedbuild.config.gnu.cross.exe.debug.1474710539.895893754">
+			<storageModule buildSystemId="org.eclipse.cdt.managedbuilder.core.configurationDataProvider" id="fr.ac6.managedbuild.config.gnu.cross.exe.debug.1474710539.895893754" moduleId="org.eclipse.cdt.core.settings" name="IKS01A2_Debug">
+				<macros>
+					<stringMacro name="XI_STMCUBE_PATH" type="VALUE_PATH_ANY" value="/Users/palantir/Work/Xively/stm32f4xx/STM32CubeExpansion_WIFI1_V2.1.1"/>
+					<stringMacro name="XI_CLIENT_PATH" type="VALUE_PATH_ANY" value="/Users/palantir/Work/Xively/stm32f4xx/xively-client-c"/>
+					<stringMacro name="XI_MEMS_STMCUBE_PATH" type="VALUE_PATH_ANY" value="/Users/palantir/Work/Xively/stm32f4xx/STM32CubeExpansion_MEMS1_V3.0.0"/>
```

Comment:
Not entire sure.  Behaviorally this is my concern and wonder if it's possible to fix:

When I'm customizing the project, in Resource->Linked Resources I set an XI_MEMS variable and an STMCUBE Wifi variable.

Then, after this, I also have to set the same path for C/C++ Build->Build Variables, which have their own XI_MEMS and XI_STMCUBE variables, as well as an XI_CLIENT_PATH.

For the TI demos we set all of these in the linked resources only, and the rest of the configuration is handled by this.